pkg/storage/testframework_test.go

```diff
@@ -194,14 +199,14 @@ func uint64ToBytes[V ~uint64](v V) []byte {
 	return b
 }
 
-func (t *TestFramework) assertPrunableSizeGreater(expected int64) {
-	require.GreaterOrEqual(t.t, float64(t.Instance.PrunableDatabaseSize()), float64(expected)*0.8)
-}
-
 func (t *TestFramework) assertPermanentSizeGreater(expected int64) {
 	require.GreaterOrEqual(t.t, float64(t.Instance.PermanentDatabaseSize()), float64(expected)*0.8)
 }
 
+func (t *TestFramework) assertPrunableSizeGreater(expected int64) {
+	require.GreaterOrEqual(t.t, float64(t.Instance.PrunableDatabaseSize()), float64(expected)*0.8)
```

Comment:
Please add a description for the function or a comment to explain the magic number.

Reply:
I did not even modify that code, I only moved it :D

Reply:
:crying_cat_face: